### PR TITLE
pageurls - fix home path

### PR DESCRIPTION
**Pull request by sardhani-ch**

### CODE REVIEW

File: src/main/types/pageURL.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ const SEPARATOR = "/";
 
 export const START = "/"; // Domain name will go here
 
-export const HOME_URL = "/home";
+export const HOME_URL = "";
```
Comment (sampankumar) [inline src/main/types/pageURL.ts:5]: Do we need this anymore?

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/types/pageURL.ts (hunk 1 above)
Comment (sardhani-ch) [reply src/main/types/pageURL.ts]: Addressed the comments

File: src/main/routes/index.ts [hunk 2]
```diff
@@ -29,8 +29,8 @@ import { soleTraderWhatIsYourRoleValidator } from "../validation/soleTraderWhatI
 
 const routes = Router();
 
-routes.get(urls.HOME_URL, indexController.get);
-routes.post(urls.HOME_URL, indexController.post);
+routes.get("", indexController.get);
```
Comment (swalke-ch) [inline src/main/routes/index.ts:32]: Can you constant instead of ""

Comment (sardhani-ch) [reply src/main/routes/index.ts]: addressed

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by swalke-ch — APPROVED** [review]

**Review by sampankumar — APPROVED** [review]

**Pull request merged**